Editor/UI/Wizard/WizardPage.cs

```diff
@@ -31,10 +34,11 @@ public WizardPage()
 
         }
 
-        public WizardPage(string name, bool allowSkip = false)
+        public WizardPage(string name, bool allowSkip = false, bool mandatory = true)
```

Comment:
What is the difference between `allowSkip` and `mandatory`?

Reply:
* Allow skip does display an additional button skip this step/page
* When the wizard is closed it checks if every mandatory pages was visited and informs about it with the isComplete variable
* Not all pages which are not mandatory (for example the about page) should show the skip button.